### ISSUE TITLE
Create page for installing packages from "Install Blockstack" button on home screen

### ISSUE THREAD

**Issue by pstan26** [blockstack/blockstack.org#236]
@guylepage3 I spoke with @muneeb-ali about creating a landing page where you can install Mac OS, Linux, (and Windows soon) packages that contain CLI, Core, and later Portal. The Docs page would no longer be where you land when clicking install.

Ideally, we were thinking it would be great to get this completed by March 7th USV event if possible so we can tell people click "install Blockstack" then they are up and running with their shell and can follow along.

I've included a back-of-the-napkin mockup of the page below using Keybase's install-page:

![screen shot 2017-02-28 at 12 35 48 pm](https://cloud.githubusercontent.com/assets/5605552/23435869/90380804-fdd6-11e6-87e8-aed89d8005bf.png)

**Comment by pstan26** [comment]
Sry screenshot would not upload for some reason. Here is a publicly viewable URL:
https://drive.google.com/open?id=0BxRgEJM6X5t-NDJYZUFhRUFoZnM

**Comment by pstan26** [comment]
Hey @guylepage3 I know this idea is coming in on short notice, but I'd be happy to focus on this with you so we have it in time for Tuesday if it can be shoehorned in. Will be in on the weekend. If you are jammed, just let me know

**Comment by guylepage3** [comment]
@pstan26 we will have to do a quick and dirty for this. v1 will need to be a headline, paragraph and download button. We can start another issue for proper requirement capturing and design that will need to be scheduled in.

**Comment by pstan26** [comment]
Hey @guylepage3 based on our conversation from today at lunch, it sounds like we can proceed with a *very* simple layout that works for desktop, tablet, phone, etc.  consisting of:

![screen shot 2017-03-02 at 4 15 05 pm](https://cloud.githubusercontent.com/assets/5605552/23527323/7c7a3970-ff63-11e6-95d7-6514e7cb8843.png)

*Headline*:
 Download Blockstack

*Body*:

- MacOS _(@shea256 once it is ready can you send the link to binary please)_
- Linux _(@jude will provide link to package shortly)_
- Windows _(greyed out)_
Source
- Github _(https://github.com/blockstack/blockstack-core)_

**Comment by jude** [comment]
(@jcnelson != @jude)

**Comment by pstan26** [comment]
fixed

**Comment by guylepage3** [comment]
@pstan26, @shea256 will be handling this issue for this sprint. I'll follow up around after in a future sprint.

**Comment by shea256** [comment]
Done with the MVP downloads page! Closing this issue.
